core/src/main/java/me/lihq/game/ScenarioBuilder.java

```diff
@@ -119,6 +120,9 @@ Set the first clues in the list to red herrings (the number of red herrings
 
             if (randHidingSpot != null) {
                 room.addClue(clue.setTileCoordinates(randHidingSpot));
+            } else
+            {
+                System.out.println("Failed to place Clue " + clue.getName() + " in room " + room.getName());
```

Comment:
I suggest this gets removed - if it's not ever being run there's no reason to keep it around. It won't be too hard to add back in for future debugging (if that ever happens)